apps/desktop/scripts/dev.mjs

```
@@ -26,6 +26,20 @@ const run = (command, args, options = {}) => {
   return child;
 };
 
+const buildOfficialConsolePlugins = () =>
+  new Promise((resolve, reject) => {
+    const child = spawn('pnpm', ['--filter', './plugins/consoles/**', 'build'], {
+      cwd: new URL('../../..', import.meta.url).pathname,
```

ESLint

[error] 'URL' is not defined.

(no-undef)